.golangci.yml

```diff
@@ -41,6 +41,12 @@ linters:
       checks:
         - all
         - -QF1008 # Excludes QF1008 from staticcheck
+    revive:
+      rules:
+        - name: package-comments
+          severity: warning
+          disabled: true
+          exclude: [ "" ]
```

Comment:
Same here; can you

- include this fix in the `ci: bump golangci from 6.5.2 to 7.0.0` commit
- move the `ci: bump golangci from 6.5.2 to 7.0.0` to be the last commit (after all the other changes)?